src/gpm-statistics.c

```diff
@@ -1681,6 +1681,12 @@ main (int argc, char *argv[])
 			  G_CALLBACK (gpm_stats_points_checkbox_stats_cb), NULL);
 
 	widget = GTK_WIDGET (gtk_builder_get_object (builder, "notebook1"));
+
+	gtk_widget_add_events (widget, GDK_SCROLL_MASK);
+	g_signal_connect (widget, "scroll-event",
+					  G_CALLBACK (gpm_dialog_page_scroll_event_cb),
```

Comment:
It gets dramatic , indents!
I do not really get why you have so heavy problems with using tabs and spaces.
And why you don't check it in your own repo in browser before pushing here.
https://github.com/tamplan/mate-power-manager/commit/7a731e4fd30210a62543f146d1757eb6924d4462

Believe me, this review isn't funny for me. I don't really like to post always the same simple issues.

Reply:
![Capture du 2019-07-10 19-43-12](https://user-images.githubusercontent.com/50292710/60991819-8e7de200-a34b-11e9-84b8-cce2fa64ab9b.png)
Sure it's not funny for me too, see the file i have, i don't want to push more modification than necessary.